### PR TITLE
DDF-2824 Updated asciidoctorj and jbake mvn plugin versions

### PR DESCRIPTION
**Pull request by mcalcote**
#### What does this PR do?
Newer versions of the asciidoctorj plugins have been released and this change points our build at them.

#### Who is reviewing it? 
@AzGoalie @vinamartin 

#### Select relevant component teams: 
@codice/docs 

#### Choose 2 committers to review/merge the PR. 
@ricklarsen @rzwiefel 

#### How should this be tested? (List steps with links to updated documentation)
Build, look for errors during the build process. Open the built documentation and confirm it appears as expected.

#### Any background context you want to provide?
There may be conflicts between our version of jbake and the older asciidoctorj plugins

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/pom.xml [hunk 1]
```diff
@@ -190,6 +190,7 @@
             <plugin>
                 <groupId>br.com.ingenieux</groupId>
                 <artifactId>jbake-maven-plugin</artifactId>
+                <version>0.0.9</version>
```
Comment (ricklarsen) [inline distribution/docs/pom.xml]: we _could_ move this version declaration to the top level pom with the others, for consistency, but this also works.

Comment (mcalcote) [reply distribution/docs/pom.xml]: Will move

**Comment by mcalcote** [comment]
build now

**Comment by mcalcote** [comment]
build now

**Pull request merged**